codesign/codesign.go

```diff
@@ -97,6 +124,11 @@ type Project interface {
 }
 
 func (m *Manager) getProject() (Project, error) {
+	// Manager is either initialised with an XCArchive or an Xcode Project.
+	if m.archive != nil {
+		return m.archive, nil
+	}
```

Comment:
Do we need a separate variable for the archive? It is not used anywhere else than in this method. I think it is the same for `project` as well. If the archive conforms to the `Project` interface then we could assign it to the project variable.

Reply:
Merged project and archive in https://github.com/bitrise-io/go-xcode/pull/141/commits/7eb170b841166e5182990b48779f8f51008a2db3